website/src/content/docs/formatter/index.mdx

```diff
@@ -93,7 +93,138 @@ const expr =
 
 There are some divergences with Prettier.
 
-### Bogus or duplicate modifiers on class properties
+### Prettier doesn't unquote some object properties that are valid JavaScript identifiers.
+
+Prettier and Biome unquote object and class properties that are valid JavaScript identifiers.
+Prettier [unquotes only valid ES5 identifiers](https://github.com/prettier/prettier/blob/a5d502513e5de4819a41fd90b9be7247146effc7/src/language-js/utils/index.js#L646).
+
+In an ecosystem where ES2015 is now widespread, this seems to be a legacy restriction.
+Thus, we decided to diverge here by unquoting all valid JavaScript identifiers in ES2015+.
+
+A possible workaround would be to introduce a configuration to set the ECMAScript version a project uses.
+We could then adjust the unquoting behaviour based on that version.
+Setting the ECMAScript version to `ES5` could match the behaviour of Prettier.
+
+```js title="example.js"
+const obj = {
+ 'a': true,
+ b: true,
+ "𐊧": true,
+}
+```
+
+Diff
+
+```js title="exmaple.js" del={4} ins={5}
+const obj = {
+  a: true,
+  b: true,
+  "𐊧": true,
+  𐊧: true,
+};
+```
+
+
+### Prettier has an inconsistent behavior for assignment in computed keys.
+
+Prettier and Biome enclose some assignment expressions between parentheses, in particular in conditionals.
+This allows to identify potentially an expression that should be a comparison.
+
+Prettier has an inconsistent behavior, because it adds parentheses for an assignment in a computed key of an object property and doesn't for a computed key of a class property, as demonstrated by the following example:
+
+Input
+
+```js title="example.js"
+a = {
+  [x = 0]: 1,
+}
+
+class C {
+  [x = 0] = 1
+}
+```
+
+Diff
+
+```js title="example.js" del={2} ins={3}
+a = {
+  [(x = 0)]: 1,
+  [x = 0]: 1,
+};
+
+class C {
+  [x = 0] = 1;
+}
+```
+
+[Playground link](https://biomejs.dev/playground?enabledLinting=false&code=YQAgAD0AIAB7AAoAIAAgAFsAeAAgAD0AIAAwAF0AOgAgADEALAAKAH0ACgAKAGMAbABhAHMAcwAgAEMAIAB7AAoAIAAgACAAIABbAHgAIAA9ACAAMABdACAAPQAgADEACgB9AAoA)
+
+To be consistent, we decide to diverge and omit the parentheses.
+Alternatively, we could enclose any assignment in a computed key of an object or of a class.
+
+
+### Prettier adds a trailing comma to type parameters of arrow functions even when it is not required.
+
+In some specific cases, a type parameter list of an arrow function requires a trailing comma to distinguish it from a JSX element.
+When a default type is provided, this trailing comma is not required.
+Here, we diverge from Prettier, because we think it better respects the original intent of Prettier, which was to add a trailing comma only when required.
```

Comment:
```suggestion
In some specific cases, a type parameter list of an arrow function requires a trailing comma to distinguish it from a JSX element.
When a default type is provided, this trailing comma is not required.
Here, we diverge from Prettier because we think it better respects the original intent of Prettier, which was to add a trailing comma only when required.
```